pkgs/tools/misc/dpt-rp1-py/default.nix

```diff
@@ -19,6 +19,7 @@ python3Packages.buildPythonApplication rec {
     pbkdf2
     pyyaml
     requests
+    setuptools
     tqdm
     urllib3
     zeroconf
```

Comment:
Can you add a pythonImportsCheck?

Reply:
Sure, although this would not have detected the problem (the build still succeeds without `setuptools`)

Reply:
Yeah, I think this is a common error that happens more often.